modules/installation-special-config-crony.adoc

```diff
@@ -44,7 +44,7 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: master
-  name: masters-chrony-configuration
+  name: 99-masters-chrony-configuration
```

Comment:
After this PR was created, It looks like the `99-` was dropped in the `cat` [line](https://github.com/openshift/openshift-docs/blob/master/modules/installation-special-config-crony.adoc#L43). If this PR gets approved for merge, the `99-` will need to be reinstated as:
`$ cat << EOF > ./masters-chrony-configuration.yaml `-> `$ cat << EOF > ./99-masters-chrony-configuration.yaml`

Reply:
It looks like the `cat` line is correct in `master` now.